[USING] rewrite and declutter daemon (#715)
zkSNACKs/WasabiDoc: docs/using-wasabi/Daemon.md

There is also a headless daemon where you do not run a resource intensive GUI, but only the command line interface.
This daemon is especially useful for power users mixing bitcoin in the backend of their servers. 

[[toc]]

---

## Available Commands

> `mix` starts Wasabi in daemon and not the GUI.

Author: ```suggestion
`wassabee` or `wassabeed` starts Wasabi wallet when the package is installed.

`dotnet run` builds the source code and starts Wasabi.

`mix` makes sure Wasabi starts in daemon and not the GUI.
```

`--wallet:` specifies the name of the hot wallet with the coins you want to CoinJoin.

`--destination:` specifies the destination wallet that the mixed coins will be coinjoined into, after the target anonymity set is reached.

> A coin will be coinjoined into the first wallet, until anonymity set target is reached, then there will be one additional CoinJoin into the `--destination` wallet.

Reviewer: ```suggestion
A coin will be coinjoined into the first wallet, until anonymity set target is reached, then there will be one additional CoinJoin into the `destination` wallet.
```

`--keepalive` keeps the daemon running after all coins have reached the anonymity set target, and continue to CoinJoin when new coins are received into the wallet.

`--help` displays help page and exit.

## Usage

### Linux

If the package is installed, execute in the command line in any directory: 

```bash
~$ wassabee mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

> If the source code is built, navigate to the Gui directory and execute:

Reviewer: ```suggestion
If the source code is built, navigate to the WalletWasabi.Gui folder (inside the cloned repo) and execute:
```

```bash
~/WalletWasabi/WalletWasabi.Gui$ dotnet run -- mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

### MacOS

If the package is installed, navigate to the Applications directory and execute: 

```bash
~/Applications/Wasabi\ Wallet.app/Contents/MacOs $ wassabee mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

> If the source code is built, navigate to the Gui directory and execute:

Reviewer: ```suggestion
If the source code is built, navigate to the WalletWasabi.Gui folder (inside the cloned repo) and execute:
```

```bash
~/WalletWasabi/WalletWasabi.Gui$ dotnet run -- mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

### Windows

If the package is installed, navigate to the WasabiWallet folder and execute:

```bash
C:\Program Files\WasabiWallet> wassabeed.exe mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

> If the source code is installed, navigate to the Gui folder and execute:

Reviewer: ```suggestion
If the source code is installed, navigate to the WalletWasabi.Gui folder (inside the cloned repo) and execute:
```

```bash

> C:\USER\WalletWasabi\WalletWasabi.Gui> dotnet run -- mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive

Reviewer: ```suggestion
\WalletWasabi\WalletWasabi.Gui> dotnet run -- mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

There is also a headless daemon where you do not run a resource intensive GUI, but only the command line interface.
This daemon is especially useful for power users mixing bitcoin in the backend of their servers. 

[[toc]]

---

## Available Commands

`wassabee` or `wassabeed` starts Wasabi wallet when the package is installed.

> 

Reviewer: ```suggestion
```

## Available Commands

`wassabee` or `wassabeed` starts Wasabi wallet when the package is installed.

> `dotnet run` builds the source code and starts Wasabi.

Reviewer: This is not a command specific to the daemon
```suggestion
```

Author: yes, but similar to `wassabee` it is needed to start Wasabi...
Not sure, should it be removed?

Author: yes, but similar to `wassabee` it is needed to start Wasabi...
Not sure, should it be removed?

Reviewer: IMO it should be removed, you are talking about available commands of the daemon and that is not one of them.
